spec/models/challenge_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+describe Challenge do
+
+  describe 'attribute_scoring' do
+    before do
+      @image = Image.new(
+        focal_length: "35mm",
+        exposure: "0.01 sec",
+        aperture: "f/2.8",
+        iso_speed: "300"
+      )
+      @correct_challenge = Challenge.new(
```

Comment:
and if u leave here instead of making variables in a before block, it's better to make them all let blocks!

let(:image) { Image.new(.....) }